specification/powerplatform/suppressions.yaml

```
@@ -0,0 +1,3 @@
+- tool: TypeSpecRequirement
+  path: powerplatform/resource-manager/Microsoft.PowerPlatform/preview/2020-10-30-preview/enterprisePolicy.json
```

```suggestion
  path: resource-manager/Microsoft.PowerPlatform/preview/2020-10-30-preview/enterprisePolicy.json
```

path is relative to suppressions.yaml